src/lib/server/orders.ts

```diff
@@ -1590,6 +1591,23 @@ async function generatePaymentInfo(params: {
 					invoiceId: invoice.r_hash,
 					processor: 'lnd'
 				};
+			} else if (isSwissBitcoinPayConfigured()) {
+				const invoice = await swissBitcoinPayCreateInvoice({
+					label,
+					orderId: `${params.orderNumber}`,
+					expiresAt: params.expiresAt,
+					toPay: {
+						amount: satoshis,
+						currency: 'SAT'
+					}
+				});
+				return {
+					address: invoice.payment_address,
+					invoiceId: invoice.invoiceId,
+					processor: 'swiss-bitcoin-pay'
+				};
+			} else {
```

Comment:
I think you can make this higher priority than LND maybe (put the if branch before lnd's)

cc @Tirodem

Reply:
I've moved SBP higher up.